src/lvl_script_commands.c

```diff
@@ -5129,6 +5129,139 @@ static void set_player_modifier_process(struct ScriptContext* context)
     }
 }
 
+static void add_to_player_modifier_check(const struct ScriptLine* scline)
+{
+    ALLOCATE_SCRIPT_VALUE(scline->command, scline->np[0]);
+    short mdfrdesc = get_id(modifier_desc, scline->tp[1]);
+    short mdfrval = scline->np[2];
+    const char *mdfrname = get_conf_parameter_text(modifier_desc,mdfrdesc);
+    if (mdfrdesc == -1)
+    {
+        SCRPTERRLOG("Unknown Player Modifier '%s'.", mdfrname);
```

Comment:
This log error does not work. The description is invalid, so the name does not work. You want `scline->tp[1]` here.